ompi/mca/osc/rdma/osc_rdma_component.c

```diff
@@ -68,9 +68,22 @@ static int ompi_osc_rdma_component_finalize (void);
 static int ompi_osc_rdma_component_query (struct ompi_win_t *win, void **base, size_t size, int disp_unit,
                                           struct ompi_communicator_t *comm, struct opal_info_t *info,
                                           int flavor);
-static int ompi_osc_rdma_component_select (struct ompi_win_t *win, void **base, size_t size, int disp_unit,
-                                           struct ompi_communicator_t *comm, struct opal_info_t *info,
-                                           int flavor, int *model);
+static int ompi_osc_rdma_component_select (struct ompi_win_t *win, void **base, size_t size, 
+                                           int disp_unit, struct ompi_communicator_t *comm,
+                                           struct opal_info_t *info, int flavor, int *model);
+
+static int ompi_osc_rdma_component_iselect (struct ompi_win_t *win, void **base, size_t size,
```

Comment:
I'm not sure this is needed - the "select" is choosing which module to use for supporting the RMA functionality. It is already a local (nonblocking) action, I think. So I'm pretty sure we don't have to split it into a begin piece and a complete piece.